packages/inventory/src/shared/TagsModal.js

```diff
@@ -69,7 +69,7 @@ class TagsModal extends React.Component {
                     }))
                 }}
                 loaded={ loaded }
-                width="auto"
+                width={'50%'}
```

Comment:
no need to use block

Reply:
Yeah! I've copied that from PF and they have it block, dumb...